cmd/earthly/secret_cmds.go

```diff
@@ -104,7 +104,7 @@ func (app *earthlyApp) secretCmds() []*cli.Command {
 	}
 }
 
-func (app *earthlyApp) actionSecretsListV2(cliCtx *cli.Context) error {
+func (app *earthlyApp) actionSecretsListV2(cliCtx *cli.Context) (err error) {
```

Comment:
Just out of curiosity. why name the returned error here? It seems like it's referenced by name in each of the return instances.

Reply:
I was on the fence about changing `path, err = app.fullSecretPath(path)` to `path, err := app.fullSecretPath(path)`; but thinking back on that, i'll revisit :)

Reply:
yeah.... not sure what I was thinking :)